### PR TITLE
Solved the task

### PR DESCRIPTION
**Pull request by Cerne13**

### CODE REVIEW

File: app/test_split_integer.py [hunk 1]
```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(32, 3)) == 32
```
Comment (VitaliyBychevoy) [inline app/test_split_integer.py:5]: add error message everywhere

**Review by VitaliyBychevoy — CHANGES_REQUESTED** [review]

File: app/test_split_integer.py (hunk 1 above)
Comment (Cerne13) [reply app/test_split_integer.py:5]: As far as I understood, we don't use parametrization here (we have separate functions in the boilerplate code).
So we have very informative tracebacks if an assertion error is thrown (with the name of the function, diff etc.).
I doubt we additionally need to do "with pytest.raises()" or "@pytest.mark.xfail" here...

I guess I'll try to pass the solution to mentors as is.

**Review by TPolina — APPROVED** [review]
Well done!

File: app/test_split_integer.py (hunk 1 above)
Comment (TPolina) [reply app/test_split_integer.py:5]: yes, in this task it is not obligatory

**Pull request closed without merging**